capture.go

```diff
@@ -40,7 +40,7 @@ type Capture struct {
 	Env []string `json:"env,omitempty"`
 
 	// Output provides the combined output from the name as a string.
```

Comment:
Trivial, but we may want to update the comment to reflect it's a []byte now or just remove the comment?